### DISCUSSION TITLE
Welcome to CMCS-DSG-DSS-Certification Discussions!

### DISCUSSION THREAD

**Discussion by nickarrow** [CMSgov/CMCS-DSG-DSS-Certification-Staging#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by donkarnag** [comment]
Hi Nick.  I have been following progress on this project and am excited to see it come along.  The CSV file that you are referencing and the link to the Excel workbook (maybe the same thing) leads to a 404 Not Found.  Are you able to share that file?

**Reply by nickarrow** [reply]
Hi there @donkarnag - we restructured the site so that all the links now work to a proper excel workbook. Back in September we were just testing out a piloting options. We're also using pages to render the site in a much cleaner look and feel. Feel free to give it a look and let us know what you think: https://cmsgov.github.io/CMCS-DSG-DSS-Certification/
